__tests__/server.test.js

```diff
@@ -152,7 +159,7 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 
-  xtest("should return a 404 with 'No comments available' if there are no comments attached to the article", () => {
+  test("should return a 404 with 'No comments available' if there are no comments attached to the article", () => {
```

Comment:
If there are no comments, is this an error? The request happened with no error, so would be a 200 and return an empty array

Reply:
Very good point, I didn't even consider that, thanks